packages/design-system/src/components/Table/Table.jsx

```diff
@@ -191,6 +193,10 @@ Table.propTypes = {
    * The viewport size at which responsive vertically stacked row styles are applied. Only used when the `stackable` prop is set to `true`. [Read more on breakpoints](/guidelines/responsive/)
    */
   stackableBreakpoint: PropTypes.oneOf(['sm', 'md', 'lg']),
+  /**
+   * Stacks the table cells on narrow screens and visually promotes the first cell of every row into a “header” for that group.
```

Comment:
What is this prop for? We didnt have it originally

Reply:
Added the StackableStyles props - specify 'header` if we want the stacked table to display with the first row as a header, with bold font and background color.

Reply:
Also just added the `stacked borderless table with headers` to the React table example

Reply:
Hm I see, its a new stacked table design then. I think for this ticket it's better to avoid adding that prop and new style to avoid expanding the scope of this work. Teams that want that style can always manually add it, or request us for it, so maybe we can tackle this in the future

Reply:
Another reason why not to include it is because our `props` arent currently setup well to handle it. We probably dont want to call the prop `stackableStyles`, and it doesnt make sense to make it `Proptype.oneOf(['header'])` if there is only one option. We need to think more intentionally on how we want folks to be using these variants and props.